source/libraries/CompileSolidity.ts

```diff
@@ -24,12 +25,12 @@ class SolidityContractCompiler {
     public async compileContracts(): Promise<CompileContractsOutput> {
         try {
             // Compile all contracts in the specified input directory
-            const inputJson: CompilerInput = await this.generateCompilerInput();
-            const compilerOutput: any = compileStandardWrapper(JSON.stringify(inputJson), this.readCallback);
-            const compileOutputJson = JSON.parse(compilerOutput);
-            if (compileOutputJson.errors) {
+            const compilerInputJson: CompilerInput = await this.generateCompilerInput();
+            const compilerOutput: any = compileStandardWrapper(JSON.stringify(compilerInputJson), this.readCallback);
+            const compilerOutputJson = JSON.parse(compilerOutput);
```

Comment:
The names here should probably be swapped.  `compileStandardWrapper` returns a JSON string, `JSON.parse(...)` turns that into a native JS object.

Reply:
Ok, or maybe the names `compilerOutput` for the first one and `compilerOutputObj` for the second would be clearest?

Reply:
That works.  I would personally go with `compilerOutputJson` for the first and `compilerOutput` for the second though.  In general, anything that is a native JSON object I refer to by the base name, and the stringified JSON form of that same object I refer to with a Json suffix.